packages/govern-contract-utils/test/erc1167-proxy-factory.test.ts

```diff
@@ -0,0 +1,120 @@
+import { expect } from 'chai'
+import { ethers } from 'hardhat'
+import { Signer } from 'ethers'
+import { defaultAbiCoder } from 'ethers/lib/utils'
+import { generateCodeFromContract } from './helpers'
+import {
+  CloneFactoryMockFactory,
+  CloneFactoryMock,
+  ClonedContract,
+  ClonedContractWithInit
+} from '../typechain'
+import
+  * as ClonedContractArtifact
+  from '../artifacts/contracts/test/ClonedContract.sol/ClonedContract.json'
+import
+  * as ClonedContractWithInitArtifact
+  from '../artifacts/contracts/test/ClonedContractWithInit.sol/ClonedContractWithInit.json'
+
+
+describe('ERC1167ProxyFactory', () => {
+  let signers: Signer[],
+    owner: string,
+    factory: CloneFactoryMock,
+    clonedContractAddress: string,
+    clonedContract: any
+
+  before(async () => {
+    signers = await ethers.getSigners()
+    owner = await signers[0].getAddress()
+    const CloneFactoryMock = (await ethers.getContractFactory('CloneFactoryMock')) as CloneFactoryMockFactory
+    factory = await CloneFactoryMock.deploy()
+  })
+
+  context('Clone', () => {
+    it('clone without constructor parameters', async () => {
+      await factory.clone()
+
+      clonedContractAddress = await factory.latestClonedContract()
+      clonedContract = (await ethers.getContractAt(
+        ClonedContractArtifact.abi,
+        clonedContractAddress,
+        signers[0]
+      )) as ClonedContract
```

Comment:
Similar comment that above, we can reuse a helper to accept types like: `getContractAt<T>`

Reply:
Feel free to create a related issue and we can do this in another PR and apply it on all tests.. I think it is better to keep the scope of a PR as small as possible otherwise some mistakes could get merged..